Gruntfile.js

```diff
@@ -55,16 +58,17 @@ module.exports = function(grunt) {
 				var file;
 				for(var i=0;i<files.length;i++) {
 					file = files[i];
-						tasks.push('exec:test:"test/' + file + '"')
+					tasks.push('exec:test:"test/' + file + '"')
 				}
-
-				console.log(tasks);
 				//tasks = ['exec:test:"test/unit/*.js"']
 				break;
 			default:
 				tasks = ['dropTestDb', 'exec:test:"test/' + type + '.coffee"', 'dropTestDb']
 				break;
 		}
+
+		console.log(tasks);
```

Comment:
Is this intended?

Reply:
yeah, I just moved this out of the case above. this is just test code so doesn't matter